### DISCUSSION TITLE
Zero query not works?

### DISCUSSION THREAD

**Discussion by abinhho** [google/mozc#1088] [Q&A]
I saw a lot of zero query lines here but I could not see it when typing.

![Screenshot 2024-10-18 at 18 01 11](https://github.com/user-attachments/assets/24b22874-1e23-4817-b3e7-7fabe8f94a7b)

![Screenshot 2024-10-18 at 18 03 07](https://github.com/user-attachments/assets/7e805140-ed6b-44ee-af65-c19f34311501)

Is there any config for disable/enable it?

**Comment by yukawa** [comment]
> I saw a lot of zero query lines here but I could not see it when typing.

Files under [src/data/zero_query](https://github.com/google/mozc/tree/master/src/data/zero_query) are intended to be used for mobile scenarios. Not seeing those entries in the desktop mode is kind of expected.

https://github.com/google/mozc/blob/5924397e162d77423c97d3a2afe3bcae7b61b6ec/src/protocol/commands.proto#L648-L656
